ThreatDragonModels/README.md

```diff
@@ -0,0 +1,25 @@
+<!--
+/*
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+-->
+
+# Threat Models
+This directory holds the threat models for this repository, which are built by [Threat Dragon](http://docs.threatdragon.org/) and stored as json files.
+
+## Threat Dragon Usage
+Threat Dragon can be used as a webapp or a desktop application.
+
+In the [webapp version](https://threatdragon.org/#/threatmodel), you will only be able to view or edit threat models in Github repositories/forks that you have push access to.
```

Comment:
❔ from a PR perspective. Any ideas on the best way to review these? It seems like the webapp can only read them after they are merged in the repo? Should the author upload the latest rendering or should reviewers find a way to put in their desktop app? Otherwise authors will have complexity of adding people to have push access?I left off on these questions when I looked at this earlier. Any suggestions would be appreciated.